### ISSUE TITLE
truediv argument of pandas.eval ignored.

### ISSUE THREAD

**Issue by simonjayhawkins** [pandas-dev/pandas#28446]
#### Code Sample, a copy-pastable example if possible

```python
>>> import pandas as pd
>>>
>>> pd.__version__
'0.25.0+350.g330bedeb7'
>>>
>>> df = pd.DataFrame({'A': range(1, 6)})
>>>
>>> df.eval('A/2')
0    0.5
1    1.0
2    1.5
3    2.0
4    2.5
dtype: float64
>>>
>>> df.eval('A/2', truediv = False)
0    0.5
1    1.0
2    1.5
3    2.0
4    2.5
dtype: float64
>>>
>>> df.eval('A/2', truediv = True)
0    0.5
1    1.0
2    1.5
3    2.0
4    2.5
dtype: float64

```
#### Problem description

The docs state..
```
truediv : bool, optional
    Whether to use true division, like in Python >= 3
```
but the `truediv` argument is ignored.

**Comment by jbrockmendel** [comment]
I'm pretty sure this is a holdover from py2

**Comment by simonjayhawkins** [comment]
i think so too. so just need to clean and add test?

possible solution https://github.com/simonjayhawkins/pandas/commit/3dd747cdaff0043b86434b53575ecded56512ea3

but would that be a breaking api change?

**Comment by jbrockmendel** [comment]
that looks like a good fix, yes.  i think it technically would be a breaking change, yes

**Comment by simonjayhawkins** [comment]
so the earliest we can introduce the deprecation is 1.0.0 and the earliest we apply the fix is 2.0.0!

**Comment by jbrockmendel** [comment]
Closed by #29812.
